leaf-ui/scripts/analysis.js

```diff
@@ -65,17 +65,16 @@ window.onload = function(){
 
 function renderNavigationSidebar(currentPage = 0){
 	clear_pagination_values();
-		
+	
 	var currentPageIn = document.getElementById("currentPage");
 	var num_states_lbl = document.getElementById("num_states_lbl");
-//	num_states_lbl.innerHTML += (analysisResult.elementList[0].valueList.length - 1);
-	num_states_lbl.innerHTML += (analysisResult.allSolution.length - 1);
+	num_states_lbl.innerHTML += (analysisResult.allSolution.length);
 	
 	currentPageIn.value = currentPage.toString();
 	
 	updatePagination(currentPage);
-	//updateNodesValues(currentPage);
-	updateSliderValues(currentPage);
+	updateNodesValues(currentPage);
```

Comment:
Since the slider doesn't exists anymore, the pagination is responsible to update the node values